manifests/machineconfigcontroller/clusterrole.yaml

```diff
@@ -2,7 +2,8 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: machine-config-controller
-  namespace: {{.TargetNamespace}}
+  labels:
+    "mco-built-in/cluster-role": ""
```

Comment:
this must be in the form `<component>.openshift.io/<functionality>` (`mco.openshift.io/something-something`)

Reply:
A lot of other operators are going to be installing roles too right?  I wonder if we should have a standard annotation for this like `openshift.io/operator-managed: ""`?

Reply:
we could yeah, Maceij suggested the above as a common standard pattern to use for labels but I'm open to a more generalized way as well

Reply:
I think we can agree on `mco.openshift.io/managed`?

Reply:
> I think we can agree on `mco.openshift.io/managed`?

I like this label. Much clearer than the previous label. 
Anyone have any opposing thoughts?

Reply:
what was the status of the generalized: `openshift.io/operator-managed: ""` bc i think that would make the most sense here if it's possible.

Reply:
I think most operators have the CVO create the resources for their operators (CRDs, RBAC, namespaces, etc) directly and doesn't need any sort of label for filtering or ownership designations. Is there a reason the MCO is different and has to do it itself rather than using the CVO?

Reply:
I did a quick `git log install/` which turned up:

```
commit 3792c944f2ced1be4fe8c2758f390d097a0b7927
Author:     Abhinav Dahiya <abhinav.dahiya@redhat.com>
AuthorDate: Tue Aug 7 12:42:10 2018 -0700
Commit:     Abhinav Dahiya <abhinav.dahiya@redhat.com>
CommitDate: Mon Aug 13 11:32:07 2018 -0700

    *: add operator sync and  manifests
    
    manifests/ -> are all the object that operator continuously manages.
    install/ -> are all the objects that need to be installed but not
    maned by the operator.
```

Which...hmm, is some sort of hint?  I think part of the idea is that we may need to specially handle upgrades for some of our objects?

I am pretty sure there's a real reason for this but it's escaping me at the moment...

Reply:
>  I think part of the idea is that we may need to specially handle upgrades for some of our objects?

Yes, CVO is designed to create and manage objects that are **required** for the operator to run correctly and not any of it's operands. So it made most sense to control objects so that MCO can at late stage perform migrations for these objects during upgrade...

Reply:
Based on this I'd personally lean towards a generic `openshift.io/operator-managed: ""` and try to propagate that to other codebases as well?

Reply:
Alrighty, I like it and it makes total sense, let's go for it, I'll change this and Steve can change the other PR accordingly